client/components/banner/index.jsx

```diff
@@ -114,7 +114,7 @@ export class Banner extends Component {
 		const { event, feature, compact, onClick, tracksClickName, tracksClickProperties } = this.props;
 
 		if ( event && tracksClickName ) {
-			this.props.recordTracksEvent( tracksClickName, {
+			this.props.recordTracksEvent?.( tracksClickName, {
```

Comment:
I wasn't able to find the root cause, but this line does indeed throw the error "this.props.recordTracksEvent is not a function". `recordTracksEvent` is passed via `mapDispatchToProps`, and is literally just the import of the action creator from the action file. It should clearly always be defined. We seem to get one or two errors per day, so the function seems to sometimes be defined -- otherwise the number of errors would be higher.